src/lib/ilp/index.js

```diff
@@ -35,14 +35,7 @@ class Ilp {
      *
      * @returns {object} - object containing the fulfilment, ilp packet and condition values
      */
-    getResponseIlp(transactionObject) {
-        const ilpData = Buffer.from(base64url(JSON.stringify(transactionObject)));
-        const packetInput = {
-            amount: this._getIlpCurrencyAmount(transactionObject.amount), // unsigned 64bit integer as a string
-            account: this._getIlpAddress(transactionObject.payee), // ilp address
-            data: ilpData // base64url encoded attached data
-        };
-
+    getResponseIlp(transactionObject, packetInput) {
```

Comment:
I think this is a breaking change. For example TTK uses this function https://github.com/mojaloop/ml-testing-toolkit/blob/c03910a87a43ca521858c264ef3ca2cb324a19ab/src/lib/mocking/middleware-functions/ilpModel.js#L58
And we don't know if there are any other repos using this. So we need to make this refactoring without breaking anything. Like leave this function like before and if you want, you can add a deprecation comment and have another new function which can serve the new functionality.

Reply:
yes, you're right.
I've updated impl. and added unit-test for this method